docs/swagger.json

```diff
@@ -224,7 +309,7 @@
                 }
             }
         },
-        "/api/mixmining/{pubkey}/report": {
+        "/api/mixmining/node/{pubkey}/report": {
```

Comment:
The `{pubkey}` parameter makes it implicit that a single `node` is involved here, I think adding it doesn't really help anything.